dm/pkg/checker/onlineddl.go

```diff
@@ -65,7 +65,7 @@ func (c *OnlineDDLChecker) Check(ctx context.Context) *Result {
 		}
 		tables := c.bwlist.Apply(realTables)
 		if len(tables) != 0 {
-			r.State = StateFailure
+			r.State = StateWarning
```

Comment:
it will be doomed to failure when meet online ddl of that table. Maybe we must keet this checker for `all` mode tasks